airflow/www_rbac/templates/airflow/graph.html

```diff
@@ -95,8 +93,9 @@
 {% block tail %}
     {{ super() }}
 
-    <script src="{{ url_for('static', filename='d3.v3.min.js') }}"></script>
-    <script src="{{ url_for('static', filename='dagre-d3.js') }}"></script>
+    <script src="{{ url_for_asset('d3.min.js') }}"></script>
+    <!--It's a custom dagre-d3 library. TODO: Fix this using NPM version-->
```

Comment:
Make this a Jinja comment, not an HTML comment -- we don't need this in the output.

Reply:
roger that!!